galette/templates/default/page.html.twig

```diff
@@ -69,6 +62,25 @@
                             </h1>
 {% if login.isLogged() %}
                         </div>
+    {% if login.isAdmin() or login.isStaff() or login.isGroupManager() %}
+        {% set documentation_path = documentation ?? '' %}
+        {% set documentation_language = ((galette_lang == 'ota' or galette_lang == 'si') ? 'en' : galette_lang == 'nb' ? 'no' : galette_lang) %}
+        {% if documentation_path is not empty %}
+                        <div class="ui compact segment">
+                            <a class="circular ui basic icon button" href="https://doc.galette.eu/{{ documentation_language }}/master/{{ documentation }}" target="_blank" data-position="left center" title="{{ _T("Read the manual") }}">
```

Comment:
I've not tested the feature yet, but I have 2 main remarks:
- pointing to `master` may lead to missing page when using a devleopment version (such as a nightly build)
- anchors are localized :/

For the first issue, that may not be a problem, but for the second, it is. The only thing we can link is a page, not a part of it

Reply:
I think I can handle that.